src/main/java/com/cryptomanager/repositories/ReportRepository.java

```diff
@@ -149,13 +150,13 @@ public int generateProjectionReport(Portfolio portfolio,int months) throws IOExc
         }
     }
     public int generateListReport(List<String> list) throws IOException{
-        LocalDateTime reportDate = LocalDateTime.now();
+        LocalDateTime.now();
```

Comment:
Não esta armazenando o tempo, logo como esta inserindo no report?
(havia esquecido de inserir no report, mas tornar essas chamadas inúteis não fazem menor sentido...)